### ISSUE TITLE
Selenium IDE option is missing in Selenium IDE Windows App

### ISSUE THREAD

**Issue by bawarim** [SeleniumHQ/selenium-ide#1800]
Hi Team,

I am using selenium IDE windows app. And during the recording I have observed that selenium IDE option is missing to perform the assertions when we do the right click on an element. 

![image](https://github.com/SeleniumHQ/selenium-ide/assets/46487726/9ca6eb0e-ec8b-437a-a55b-5de01013d47c)

While, the same is available in browser extension. Could you please help? 

![image](https://github.com/SeleniumHQ/selenium-ide/assets/46487726/5a509b7f-32c1-49f6-a682-370b2dee18c8)

**Comment by toddtarsi** [comment]
@bawarim - Looking at this screenshot, I see the builtin browser right click stuff and just the label for Selenium IDE in browser. What in particular are you looking for? Assert variants of the wait for stuff?

**Comment by bawarim** [comment]
Yes, when we do right click on browser then we get Selenium IDE option which we can select to choose from list of different assertions. 
![image](https://github.com/SeleniumHQ/selenium-ide/assets/46487726/32d17db7-8eeb-4fe3-af63-1b9958555cb1)

But In selenium IDE windows app, I can not find this kind of shortcut to perform assertions.

**Comment by bawarim** [comment]
@toddtarsi Any input on this?

**Comment by toddtarsi** [comment]
@bawarim - Oh yeah, this will be no issue! I only kept the wait for checks because they're so much more resilient, but if people want access to asserts, I'll add em too. Do we need verify? I never knew the difference between an assert and a verify tbh

**Comment by toddtarsi** [comment]
@bawarim - Also, you should try the latest beta! It's a much more intuitive using experience IMO
